api/ws/workers/peersUpdateRules.js

```diff
@@ -77,7 +88,10 @@ PeersUpdateRules.prototype.internal = {
 		//ToDo: Peer headers and connectionId validation
```

Comment:
good catch, so this ToDo is solved by applying handshake before update, so not needed anymore